<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <meta content="" name="description"/>
    <meta content="" name="author"/>
    <!-- For iPhone -->
    <meta name="msapplication-TileColor" content="#4076e0">
    <!-- Chrome, Firefox OS and Opera -->
    <meta name="theme-color" content="#4076e0">
    <!-- Windows Phone -->
    <meta name="msapplication-navbutton-color" content="#4076e0">
    <title>Virtual Labs</title>
    <link rel="shortcut icon" href="./assets/images/favicon.ico" />
    <link rel="stylesheet" href="./assets/css/bootstrap.min.css" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/4.0.0/github-markdown.min.css" integrity="sha512-Oy18vBnbSJkXTndr2n6lDMO5NN31UljR8e/ICzVPrGpSud4Gkckb8yUpqhKuUNoE+o9gAb4O/rAxxw1ojyUVzg==" crossorigin="anonymous" />
    <link
      href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Raleway&display=swap"
      rel="stylesheet"
    />
    <link
      rel="stylesheet"
      href="./assets/fonts/font-awesome-4.7.0/css/font-awesome.min.css"
    />
    <link rel="stylesheet" href="./assets/css/github-markdown.min.css" />
    <link rel="stylesheet" href="./assets/css/vlabs-style.css" />
    
    <script src="./assets/js/jquery-3.4.1.slim.min.js"></script>
    <script
      src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.5.4/dist/umd/popper.min.js"
      integrity="sha384-q2kxQ16AaE6UbzuKqyBE9/u/KzioAlnx2maXQHiDX9d4/zp8Ok3f+M7DPm+Ib6IU"
      crossorigin="anonymous"
    ></script>
    <script
      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.min.js"
      integrity="sha384-pQQkAEnwaBkjpqZ8RU1fF1AKtTcHJwFl3pblpTlHXybJjHpMYo79HY3hIi4NKxyj"
      crossorigin="anonymous"
    ></script>
    <style>
      h1 {
      font-size: 1.8rem;
      }
      h2 {
      font-size: 1.4rem;
      }
      h3 {
      font-size: 1.2rem;
      }

      h2, h3 {
      font-weight: 800;
      }
    </style>
  </head>
  <body class="p-0 container-fluid vlabs-page d-flex flex-column justify-content-between">
    <header class="vlabs-header sticky-top bg-white">
      <nav class="p-0 navbar navbar-light d-flex align-items-stretch">
        
        <button class="navbar-toggler px-4"
        	    type="button" onclick="toggle()">
          <span class="navbar-toggler-icon"></span>
        </button>
    
        <div class="d-flex justify-content-center">        
          <a href="http://vlab.co.in" class="navbar-brand">
            <img src="./assets/images/vlabs-color-small-moe.jpg" alt="vlabs logo">
          </a>
        </div>
        
        <div id="headerNavbar" class="border-top flex-grow-1 d-flex align-items-center">
          <ul class="navbar-nav ml-auto text-center d-flex flex-md-row">
            <li class="nav-item px-2 m-1">
              <a href="http://vlab.co.in">HOME</a>
            </li>
            <li class="nav-item px-2 m-1">
              <a href="http://vlab.co.in/participating-institutes">PARTNERS</a>
            </li>
            <li class="nav-item px-2 m-1">
              <a href="http://vlab.co.in/contact-us">CONTACT</a>
            </li>
          </ul>
        </div>
      </nav>
    </header>
    <div class="container-fluid flex-fill d-flex flex-column vlabs-page-main">
      <div class="row flex-grow-1 px-4 d-flex flex-nowrap">
        <div class="nav flex-column sidebar flex-shrink-0" id="menu">
            <div>
              <div class="d-flex">
                <a href="index.html" class="p-2 ">
                  Aim
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="theory.html" class="p-2  current-item ">
                  Theory
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="pretest.html" class="p-2 ">
                  Pretest
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="procedure.html" class="p-2 ">
                  Procedure
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="simulation.html" class="p-2 ">
                  Simulation
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="posttest.html" class="p-2 ">
                  Posttest
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="references.html" class="p-2 ">
                  References
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="feedback.html" class="p-2 ">
                  Feedback
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
        </div>        <div class="vlabs-page-content px-4 pb-4 flex-grow-1 markdown-body">
          <div class="text-center">
            <h2 id="calculating-genetic-distances-between-protein-sequences">Calculating Genetic Distances between Protein Sequences</h2>
        
          </div>
            <p>Sequence alignment is most interesting active area in the field of bioinformatics. An important step in the analysis of nucleotide or amino acid sequence alignment is calculation of the genetic distances or evolutionary distances matrix between the pair sequences. It has been noted that evolutionary changes act of DNA sequence and the sequence changes happens in the time course. Two sequences deriving from a common ancestor independently evolve and diverge later, and the measure of this divergence is termed as genetic distance and plays many roles in sequence analysis in bioinformatics and molecular biology. Genetic distances is linearly proportional to the elapsed time and measures sequence similarity. Estimating evolutionary distances between biological sequences is important to construct phylogenetic tress and understanding gene and protein evolution strategies. Considering evolutionary tree for sequences, length of the branch denotes distances between the sequences (nodes) in the tree and it has been noted that genetic divergence provides evolutionary tree relating to particular sequences. 
        &nbsp;</p>
        <p>There are various computational approaches available to predict structural and functional properties of biological sequence. Generally two types of sequence alignment were performed namely pairwise sequence alignment (PSA) and multiple sequence alignment (MSA). Pairwise alignment considers comparing two sequences at a time while multiple sequence alignment focuses on aligning more than two related sequences that considers multiple sequence alignment more useful compared to pairwise alignment as it aligns multiple members of a sequence family and give access to more biological information. This indicated multiple sequence alignment as a prerequisite to measure genomic analysis for identification of conserved regions and functional motifs and ancestor sequence profiling. Sequence alignment of amino acids were at highest priority compared to nucleotide level alignments, as proteins are the building blocks and functional biological molecules that carry structural and functional information and strongly connected to structural biology aspects. Considering the multidisciplinary approach, it has been reported in literature as multiple sequence alignment provides an open window for studying evolutionary relationships, functional properties, and structural perspective of biological macromolecules in a more concise manner.  Scoring matrices are calculated in sequence alignments to identify similarity and dissimilarities between the biological sequences of interest. In amino acid sequence alignment, similarity score is counted along with identity score which denotes the amino acids sharing similar physicochemical properties. PAM and BLOSSUM were the substitution matrices employed for protein sequence alignment. 
        &nbsp;</p>
        <p>In multiple sequence alignment, a set of homologous amino acid or nucleotide sequences are arranged in a matrix in which column represents homologous characters and functionally related protein structure. Multiple sequence alignment performance depends on its ability to assess reference alignment producing biological information regarding conserved structures. Obtained output score can be compared with score of reference alignment, known as Sum-of-Pairs (SP) score, with fraction of residue pairs in reference alignment or by Total Column (TC) score, describing the identified fraction of reference columns. While aligning the sequence, the ‘Symmetrized SP’(SSP) recoding ignores the gaps and treats them as blanks in an alignment. The name indicates the similarities of the existing SP method for comparing alignments. Seq indicates the recoding and provides a simple record of gap information and treats all gaps in a sequence equally. Consecutive gap positions were the product of a single insertion or deletion mutation. pos recoding incorporates the positional information gap occurance in a sequence, but without provide the details of  temporal phylogenetic location of the gap produced. evol recoding includes necessary information of pos recoding, and also incorporates indel event leading to that gap occurs in a phylogenetic tree. It is noted that the variation in alignment quality have a significant impact on the genetic diversity that is calculated between the same pairs of sequences.</p>
        <p>&nbsp;</p>
        <p><strong>Genetic Distance Estimation Using R</strong>
        &nbsp;</p>
        <p>Phylogenetic analyses on a genomic scale to addresses the issues ranging from the prediction of gene and protein function to organismal relationships. In a broader perspective, computing the relatedness of organisms either by phylogenetic gene by gene analyses or phylogenomic whole genome comparison methods reveals high-quality results for demonstrating phylogenies. In bioinformatics, Phylip (Phylogeny Inference Package) software is a free package of programs for inferring phylogenies of living species and organisms. </p>
        <p>&nbsp;
        <strong>Calculating genetic distances between protein sequences</strong>
        &nbsp;</p>
        <p>Using R programming, we can calculate the genetic distances between protein sequences using the “dist.alignment()” function in the SeqinR package. The dist.alignment() function takes a multiple alignment as input. Based on the multiple alignment t, dist.alignment() calculates the genetic distance between each pair of proteins in the multiple alignment.</p>
        <p>&nbsp;
        Consider example, Uniprot accession number P06747,  rabies virus phosphoprotein, P0C569, Mokola virus phosphoprotein, O56773,  Lagos bat virus phosphoprotein and Q5VKP1 is Western Caucasian bat virus phosphoprotein. Based on the genetic distance matrix above, it is noted that  genetic distance between Lagos bat virus phosphoprotein (O56773) and Mokola virus phosphoprotein (P0C569) is  about 0.414 and is the smallest value. Similarly, the genetic distance between Western Caucasian bat virus phosphoprotein (Q5VKP1) and Lagos bat virus phosphoprotein (O56773) is about 0.507,the biggest. The larger the genetic distance between two sequences, the more amino acid changes or indels that have occurred since they shared a common ancestor, and the longer ago their common ancestor probably lived (Fig.1).</p>
        <p> <img src="https://user-images.githubusercontent.com/79529753/127532312-5565ef68-afff-4ac9-bd65-2bea9746a300.png" alt="image"></p>
        <p>&nbsp;
        Multiple sequence alignment for finding genetic distance between proteins of multiple organisms</p>
        
        </div>
      </div>
    </div>
    <footer class="text-light pt-2 vlabs-footer d-flex flex-column">
      <div class="row px-5">
        <div class="col d-flex flex-column">
          <span class="font-weight-bold vlabs-footer-sect-name pb-2 mb-3">Community Links</span>
          <a href="http://www.sakshat.ac.in/" class="text-light">Sakshat Portal</a>
          <a href="http://outreach.vlabs.ac.in/" class="text-light">Outreach Portal</a>
          <a href="http://vlab.co.in/faq" class="text-light">FAQ: Virtual Labs</a>
        </div>
        <div class="col d-flex flex-column">
          <span class="font-weight-bold vlabs-footer-sect-name pb-2 mb-3">Contact Us</span>
          <span> Phone: General Information: 011-26582050 </span>
          <span> Email: support@vlabs.ac.in </span>
        </div>
        <div class="col d-flex flex-column">
          <span class="font-weight-bold vlabs-footer-sect-name pb-2 mb-3">Follow Us</span>
          <div class="social-links">
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #55acee;" href="https://twitter.com/TheVirtualLabs">
    	  <i class="fa fa-twitter"></i>
    	</a>
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #3b5998;" href="https://www.facebook.com/Virtual-Labs-IIT-Delhi-301510159983871/">
    	  <i class="fa fa-facebook"></i>
    	</a>
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #e52d27;" href="https://www.youtube.com/watch?v=asxRaOgk6a0">
    	  <i class="fa fa-youtube"></i>
    	</a>
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #2867B2;" href="https://in.linkedin.com/in/virtual-labs-008ba9136">
    	  <i class="fa fa-linkedin"></i>
    	</a>
          </div>
        </div>
      </div>
      <div class="m-0 py-2 text-center" style='font-family: "Open Sans", sans-serif; background: #212121;'>
        <a class="text-primary font-weight-bold" href="https://www.gnu.org/licenses/agpl-3.0.en.html"> AGPL 3.0 </a>
        &nbsp;&amp;&nbsp;
        <a class="text-primary font-weight-bold" href="https://creativecommons.org/licenses/by-nc-sa/4.0/"> Creative Commons (CC BY-NC-SA 4.0) </a>
      </div>
    </footer>

      <script src="./assets/js/toggleSidebar.js"></script>
      
  </body>
</html>
